util/arbmath/moving_average_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbmath
+
+import "testing"
+
+func TestMovingAverage(t *testing.T) {
+	ma := NewMovingAverage[int](5)
+	if ma.Average() != 0 {
+		t.Errorf("moving average should be 0 at start, got %v", ma.Average())
```

Comment:
nit: Capitalize, print got before want, and use %d for ints. Same below.

```
if avg := ma.Average(); avg != 0 {
    t.Errorf("Average() = %d, want 0", avg)
}
```


https://google.github.io/styleguide/go/decisions.html#got-before-want

Reply:
Why use %d?

Reply:
That's generally the convention for printing decimal integers.

https://pkg.go.dev/fmt